### PR TITLE
Use exact Deno URLs

### LINKED ISSUE #341: feat: Have the latest grammY version in all examples
We could create a plugin that build the most recent grammY version into the docs by using the `@GRAMMY_VERSION` syntax in all URLs and other places. The version can easily be obtained from https://cdn.deno.land/grammy/meta/versions.json.

This would require building a new version of the docs whenever a new release is done. However, given how often the docs are updated, I think that this will not become a problem.

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Closes #341.

### CODE REVIEW

**Review by Loskir — APPROVED** [review]
👏

File: site/docs/guide/introduction.md [hunk 1]
```diff
@@ -57,8 +57,6 @@ _________        sends a         ____________                    ____________
         Telegram's responsibility                             your responsibility
 ```
 
-<!-- TODO: create a proper diagram instead of the ASCII art -->
```
Comment (roj1512) [inline site/docs/guide/introduction.md:60]: Is this intended?

Comment (KnorpelSenf) [reply site/docs/guide/introduction.md:60]: Yes. At some point I decided that ASCII art is not much worse than images, but it comes with several advantages:
- faster to create
- faster to modify
- needs less tooling
- doesn't require graphics skills from contributors
- works on all screen sizes due to proper scrolling

Comment (KnorpelSenf) [reply site/docs/guide/introduction.md:60]: - much easier to translate

**Review by roj1512 — APPROVED** [review]

File: site/docs/zh/guide/introduction.md [hunk 2]
```diff
@@ -57,8 +57,6 @@ _________    telegram     ____________                ____________
         Telegram 负责的部分                                        你负责的部分
 ```
 
-<!-- TODO:  创建一个适当的图表，而不是 ASCII 图表 -->
```
Comment (KnorpelSenf) [inline site/docs/zh/guide/introduction.md:60]: It's a bit funny that this was even translated :D

**Pull request merged**